### PR TITLE
Genio 3: Fix 32 bit build

### PR DESCRIPTION
**Pull request by jackburton79**
Fix 32 bit build:

N.B: I left the liblexilla version check for now. Do you want me to remove it ?

### CODE REVIEW

**Comment by korli** [comment]
Please remove it, haikuporter will add it automatically.

**Comment by Begasus** [comment]
LGTM

File: haiku-apps/genio/genio-3.0.recipe [hunk 1]
```diff
@@ -32,14 +32,14 @@ REQUIRES="
 	gcc${secondaryArchSuffix}_syslibs_devel
 	cmd:clang >= 16
 	lib:libeditorconfig$secondaryArchSuffix
-	lib:liblexilla$secondaryArchSuffix >= 5.2.4
+	lib:liblexilla$secondaryArchSuffix
 	lib:libgit2$secondaryArchSuffix
```
Comment (korli) [inline haiku-apps/genio/genio-3.0.recipe:36]:
```suggestion
	lib:libgit2$secondaryArchSuffix
	lib:liblexilla$secondaryArchSuffix
```

File: haiku-apps/genio/genio-3.0.recipe [hunk 2]
```diff
@@ -32,14 +32,14 @@ REQUIRES="
...
 	lib:libyaml_cpp$secondaryArchSuffix
 	"
 
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
-	devel:libeditorconfig
+	devel:libeditorconfig$secondaryArchSuffix
 	devel:liblexilla$secondaryArchSuffix >= 5.2.4
 	devel:libgit2$secondaryArchSuffix >= 1.5
```
Comment (korli) [inline haiku-apps/genio/genio-3.0.recipe:44]:
```suggestion
	devel:libgit2$secondaryArchSuffix >= 1.5
	devel:liblexilla$secondaryArchSuffix >= 5.2.4
```

**Comment by jackburton79** [comment]
done

**Comment by Begasus** [comment]
Thanks (again) :) +1

**Pull request merged**